packages/components/src/components/code-snippet/_code-snippet.scss

```diff
@@ -202,16 +202,16 @@
     position: relative;
     order: 1;
     min-height: rem(56px);
-    max-height: rem(238px);
+    max-height: 100%;
     overflow: hidden;
     transition: max-height $duration--moderate-01 motion(standard, productive);
   }
 
   // expanded snippet container
   .#{$prefix}--snippet--multi.#{$prefix}--snippet--expand
     .#{$prefix}--snippet-container {
-    max-height: 100%;
     padding-bottom: $spacing-05;
+    overflow-y: auto;
```

Comment:
Can we move this to L206?

Reply:
Sure can - apply it for both closed and expanded modes